src/NuGet.Core/NuGet.Common/Telemetry/NuGetTelemetryService.cs

```diff
@@ -0,0 +1,38 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+
+namespace NuGet.Common
+{
+    /// <summary>
+    /// Base class for nuget telemetry services
```

Comment:
>Base [](start = 8, length = 4)

I don't see what inherits this. What's the purpose of this class? Can't the caller just new up a telemetry event on their own?